app/controllers/maintainers_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class MaintainersController < ApplicationController
+  before_action :load_project
+  def edit
+  end
+
+  def create
+    user = User.find_by(id: params[:maintainer_id])
+    @project.maintainers << user
+    redirect_to edit_project_maintainers_path
+  end
+
+  def destroy
+    user = User.find_by(id: params[:maintainer_id])
+    @project.maintainers.destroy(user)
```

Comment:
_disclaimer: nem fogom csak azért visszadobni, meg gondold végig, hogy neked is segítene-e ez a kódstílus. nem kötelező így csinálni, csak egy ötlet_

`@project.maintainers.destroy(egyszer_használatos_változó_amit_nem_szeretek)`

helyette

`@project.maintainers.find_by(id: params[:id]).destroy`

Reply:
vigyázz! @Gregurevic két sora nem ugyanazt csinálja. az első kitöröl egy kapcsolatot két objektum között, a második kitöröl egy konkrét usert!

Reply:
Uhh, bocsi. Itt tényleg butaságot írtam.